perceval/backends/core/github.py

```diff
@@ -569,7 +569,10 @@ def issue_comments(self, issue_number):
         return self.fetch_items(path, payload)
 
     def issues(self, from_date=None):
-        """Get the issues from pagination. Note that issues contain also pull requests."""
+        """Get the issues from pagination. Note that issues contain also pull requests.
+        
+        You can filter them by using the optional arguments.
```

Comment:
The docstring should be improved. A docstring should be composed of:
- a first line that summarizes what the method does
- a body that explains what the method does (optiona)
- a description for each param
- a description of what the method returns (optional)
- a description of the errors that the method raise (optional)

As examples, please have a look at the methods [show](https://github.com/chaoss/grimoirelab-perceval/blob/master/perceval/backends/core/git.py#L1041),  [fetch_items](https://github.com/chaoss/grimoirelab-perceval/blob/master/perceval/backends/core/git.py#L134) and [has_archiving](https://github.com/chaoss/grimoirelab-perceval/blob/master/perceval/backends/core/git.py#L167) in git.py

Reply:
Ok, cool. I learned something which is very important I guess. 
Sorry, I didn't know this before. 

I can use them in my microtasks as well. Thanks for sharing the information. :slightly_smiling_face: